quodlibet/quodlibet/ext/events/waveformseekbar.py

```diff
@@ -190,23 +191,28 @@ def reset(self, rms_vals, player):
         self.queue_draw()
 
     def draw_waveform(self, cr, width, height, elapsed_color, remaining_color):
-        line_width = CONFIG.line_width
+        hidpi = self.get_scale_factor() == 2
```

Comment:
I have no idea how GTK people will make this (integer) property evolve in the future. I'll test for > 1 then.